packages/components/fields/async-creatable-select-field/src/async-creatable-select-field.visualroute.jsx

```diff
@@ -77,15 +77,27 @@ const DefaultRoute = () => (
         isReadOnly={true}
       />
     </Spec>
-    <Spec label="when has warning">
+    <Spec label="with warning when not touched">
+      <AsyncCreatableSelectField
+        title="State"
+        value={value}
+        onChange={() => {}}
+        loadOptions={loadOptions}
+        horizontalConstraint={7}
+        warnings={{ customWarning: true }}
+        renderWarning={() => 'Custom warning'}
+      />
+    </Spec>
+    <Spec label="with warning when touched">
       <AsyncCreatableSelectField
         title="State"
-        name="form-field-name"
         value={value}
         onChange={() => {}}
         loadOptions={loadOptions}
         horizontalConstraint={7}
-        hasWarning={true}
```

Comment:
@CarlosCortizasCT Do you think we should delete the `<Spec>`s with the legacy `hasWarning` or leave them?

Reply:
@CarlosCortizasCT ?

Reply:
@kark I think we can remove them when we are finally removing everything about the hasWarning prop.

Reply:
Yes, I think we should keep them to make sure we don't break that behaviour ahead of time.
We still want to support that prop although we want to encourage consumers to not use it anymore.

Reply:
Okay. @CarlosCortizasCT I do not have the permission to either approve the percy test or merge this PR. So I believe someone on the UI-Kit will have to do both

Reply:
> Many thanks for your work in this PR, Chisom 🙇
> 
> Please, wait for Filip and another Shield engineer approval 🙏

As I explained before, we need another Shield engineer approval as well as @FilPob's one.

Reply:
> @kark I think we can remove them when we are finally removing everything about the hasWarning prop.

My point here was that we are removing a bunch of `<Specs>` in this PR e.g.:
<img width="1858" alt="Screenshot 2024-01-29 at 16 43 04" src="https://github.com/commercetools/ui-kit/assets/49066275/3e894df1-99e5-4d76-9373-a3a47c8e7d36">
and we should probably leave them for now

Reply:
Ahhhhh Okay. I get it now. Will add them back plus the new ones